### ISSUE TITLE
Unable to import module in the same directory

### ISSUE THREAD

**Issue by jomo** [SublimeLinter/SublimeLinter-pylint#6]
My file structure looks like this:

```
┬ pylint_test
├── modulea.py
└── moduleb.py
```

modulea.py:

``` python
class ModA():
    def __init__(self):
        self.foobar = "foobar"
    def foob(self):
        return self.foobar
```

moduleb.py:

``` python
import modulea # this line has: `Error: Unable to import 'modulea'`

foobar = modulea.ModA()
print foobar.foob()
```

I did not touch the config and I'm not using a pylintrc file.

Running pylint via commandline doesn't show this error, so I'm quite certain it's an issue with this plugin and not pylint:

``` bash
$ pylint pylint_test/moduleb.py

************* Module moduleb
C:  4, 0: Final newline missing (missing-final-newline)
C:  1, 0: Missing module docstring (missing-docstring)
C:  3, 0: Invalid constant name "foobar" (invalid-name)

Report
======
4 statements analysed.

External dependencies
---------------------
::

    modulea (moduleb)
[...]
```
